src/main/kotlin/com/quipalup/katydid/logentry/primaryadapter/httprest/CreateLogEntryEndpoint.kt

```diff
@@ -0,0 +1,31 @@
+package com.quipalup.katydid.logentry.primaryadapter.httprest
+
+import com.quipalup.katydid.common.domain.Id
+import com.quipalup.katydid.logentry.domain.LogEntry
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class CreateLogEntryEndpoint {
+    @PostMapping("/log-entries")
+    @ResponseStatus(HttpStatus.CREATED)
+    fun execute(@RequestBody newLog: LogEntry): MutableList<LogEntryResource> {
+
+        val log = LogEntryResource(
+            id = Id(),
+            type = JsonApiTypes.MEAL_LOG_ENTRY,
+            attributes = LogEntry(
+                time = newLog.time,
+                description = newLog.description,
+                amount = newLog.amount,
+                unit = newLog.unit
+            )
+        )
+            val logEntries: MutableList<LogEntryResource> = ArrayList()
+            logEntries.add(log)
+            return logEntries
```

Comment:
try to resolve this comment after resolving the others. Now try to figure out which code you hae to write down in this section to make the CT pass. It has not to be the final code that will be run in production, it has to be enough to be sure that we are going in the right direction. After this, we'll discuss how to create a log entry in domain.